src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -74,6 +76,48 @@ public async Task ProcessPostActionShouldSucceed()
             await Context.Received(1).SetVariableAsync(settings.ResponseBodyVariable, "Some result", Arg.Any<CancellationToken>());
         }
 
+        [Theory]
+        [InlineData("http://169.254.169.254/")]
+        [InlineData("http://[fd00:ec2::254]/")]
+        [InlineData("http://168.63.129.16/")]
+        [InlineData("http://10.0.0.0/")]
+        [InlineData("http://172.16.0.0/")]
```

Comment:
this a public repository...
avoid put sensible datas.
you can set fakes data

Reply:
Done!